### DISCUSSION TITLE
Vercel Dashboard Page Crash

### DISCUSSION THREAD

**Discussion by adeludedperson** [vercel/community#2986] [Bug Report]
### Page to Investigate

https://vercel.com/dashboard

### Steps to Reproduce

After Going to the dashboard.

Open Command Menu From Footer.

Click on Create New Team.

After 10 - 15 seconds hang.

Your Page Crashes.

Tried in Incognito Also.

Video:
https://github.com/vercel/community/assets/83860384/d2036db4-27c8-4af1-b303-df1c8f3a18a2

**Comment by smaeda-ks** [comment]
thanks, we're looking into this

**Reply by smaeda-ks** [reply]
Confirmed the issue on our side too. We will let you know once it's fixed. Thanks for flagging this

**Reply by smaeda-ks** [reply]
This has been fixed

**Reply by adeludedperson** [reply]
Thanks
